include/utils.hpp

```diff
@@ -15,6 +16,16 @@
 #include <typeinfo>
 #endif
 
+#ifndef DISABLE_SIMD
+#define DISABLE_SIMD 0
```

Comment:
Do we want to be able to disable SIMD via macros?

Reply:
Don't know. It's just a very effective tool for testing the effect of `stdx::simd`.